index-pl.html

```diff
@@ -41,8 +41,8 @@
     <p5 src="js/sketch0.js"></p5>
     <div class="overlay">
       <div>
-        <h1>The Multiplicative Power of Masks</h1>
-        <h3>An Explorable Essay on How Masks Can End COVID-19</h3>
+        <h1>Zwielokrotniona Moc Maseczek</h1>
```

Comment:
W poprzednich fragmentach tłumaczeń używaliśmy "maski" zamiast "maseczki", więc może użyjmy tej formy też tu i poniżej?
Chociaż zastanawiam się jeszcze której formy finalnie użyć : )

Reply:
z tego zo widze to rózne portale używaja zamiennie. 
maseczki dla mnie brzmi lepiej, bo są też przyłbice (które nie są zbyt efektywne, a niektórzy nazywająje maskami)